projects/npm-tools/packages/npm-scripts/src/config/npmscripts.config.js

```diff
@@ -7,7 +7,7 @@ const path = require('path');
 
 const CHECK_AND_FIX_GLOBS = [
 	'/*.{js,ts}',
-	'/{dev,src,test}/**/*.{js,scss,ts,tsx}',
+	'/{dev,src,test,extra}/**/*.{js,scss,ts,tsx}',
```

Comment:
Can you sort the directory names alphabetically? `dev,extra,src,test`

Reply:
Ah, the classic, brings back memories.